models/Note.js

```diff
@@ -90,12 +90,13 @@ class Note extends BaseModel {
     const props = _.pick(note, ['noteType', 'content', 'selector', 'json'])
 
     // $FlowFixMe
-    const path = urlparse(note.inReplyTo).path.substr(45)
+    // const path = urlparse(note.inReplyTo).path.substr(45)
     props.selector = note['oa:hasSelector']
-    const document = await Document.byPath(urlToId(note.context), path)
-    props.documentId = urlToId(document.id)
-    props.publicationId = note.context.id || note.context
```

Comment:
These lines we need to keep. 
Notes sent by the users of the api will not have a documentId or a publicaitonId. They will have a context and a inReplyTo